### PR TITLE
Update step.yml

### PR DESCRIPTION
**Pull request by Mithrandir21**
Updated wording in Module and Variant fields to clarify how to test all modules and variants.

### Checklist

- [X] I've read and accepted the [Contribution Guidelines](https://github.com/bitrise-steplib/.github/blob/main/CONTRIBUTING.md)
- Requires _MAJOR / MINOR / PATCH_ [version update](https://semver.org/)

### Context

Description clarification needed to figure out how to test all Modules and Variants, specifically in a multi-module environment.

### Changes

- Simply added clarification to the end of the Module and Variant descriptions.

### CODE REVIEW

File: step.yml [hunk 1]
```diff
@@ -34,17 +34,17 @@ inputs:
     opts:
       title: Module
       summary: |
-        Set the module that you want to test. To see your available modules, please open your project in Android Studio and go to [Project Structure] and see the list on the left.
+        Set the module that you want to test. To see your available modules, please open your project in Android Studio and go to [Project Structure] and see the list on the left. Leave blank to test all modules.
```
Comment (BanyikAnna) [inline step.yml]: Set the module that you want to test. To see your available modules, please open your project in Android Studio, go to **Project Structure** and see the list on the left. Leave blank it to test all modules.

PS: not sure what we leave blank? the list or is it a field we have to leave blank? Up to you but we need what we leave blank ;)

File: step.yml [hunk 2]
```diff
@@ -34,17 +34,17 @@ inputs:
...
       description: |
-        Set the module that you want to test. To see your available modules, please open your project in Android Studio and go to [Project Structure] and see the list on the left.
+        Set the module that you want to test. To see your available modules, please open your project in Android Studio and go to [Project Structure] and see the list on the left. Leave blank to test all modules.
```
Comment (BanyikAnna) [inline step.yml]: Set the module that you want to test. To see your available modules, please open your project in Android Studio, go to **Project Structure** and see the list on the left. Leave blank it to test all modules.

PS: not sure what we leave blank? the list or is it a field we have to leave blank? Up to you but we need what we leave blank ;)

File: step.yml [hunk 3]
```diff
@@ -34,17 +34,17 @@ inputs:
...
       is_required: false
   - variant: ""
     opts:
       title: Variant
       summary: |
-        Set the variant that you want to test. To see your available variants, please open your project in Android Studio and go to [Project Structure] -> variants section.
+        Set the variant that you want to test. To see your available variants, please open your project in Android Studio and go to [Project Structure] -> variants section. Leave blank to test all variants.
```
Comment (BanyikAnna) [inline step.yml]: Set the variant that you want to test. To see your available variants, please open your project in Android Studio, go to **Project Structure**, then to the **variants** section. Leave blank to test all variants.

PS: not sure what we leave blank? the list or is it a field we have to leave blank? Up to you but we need what we leave blank ;)

File: step.yml [hunk 4]
```diff
@@ -34,17 +34,17 @@ inputs:
...
       description: |
-        Set the variant that you want to test. To see your available variants, please open your project in Android Studio and go to [Project Structure] -> variants section.
+        Set the variant that you want to test. To see your available variants, please open your project in Android Studio and go to [Project Structure] -> variants section. Leave blank to test all variants.
```
Comment (BanyikAnna) [inline step.yml]: Set the variant that you want to test. To see your available variants, please open your project in Android Studio, go to **Project Structure**, then to the **variants** section. Leave blank to test all variants.

PS: not sure what we leave blank? the list or is it a field we have to leave blank? Up to you but we need what we leave blank ;)

**Review by godrei — APPROVED** [review]

**Comment by godrei** [comment]
@Mithrandir21 , thanks for the contribution!

**Pull request merged**